verifier/src/test/groovy/com/okta/jwt/impl/OktaJWTClaimsVerifierTest.groovy

```diff
@@ -180,8 +204,51 @@ class OktaJWTClaimsVerifierTest {
                 iss: "invalid_testIssuer"
         ])
 
-        TestSupport.expect(BadJWTException) {
+        expect(BadJWTException) {
             verifier.verify(claimsSetId, idTokenContext)
         }
     }
+
+    @Test
+    void IdTokenDecodeNoSubjectTest() {
+
+        JWTClaimsSet claimsSet = new JWTClaimsSet(basicValidClaims + [
+                aud: ["my_clientId"],
```

Comment:
We should only be validating against a single expected audience, even if the token can be used for others.